### ISSUE TITLE
Need some example code.

### ISSUE THREAD

**Issue by brilliantwf** [Daniel-t/node-red-contrib-aws#58]
Thank you for making such great code, I am using AWS services now, I would like to  use some of your code to show some of AWS's services, but without the sample code, could you do some simple how to use the code? Or how to modify the payload or functionality that Node-red can use through the AWS API. thank you very much.

**Comment by sxwebster** [comment]
I have to agree with this. Basic functions like taking a file and uploading to AWS S3 is actually quite difficult to understand.

**Comment by johnebgood** [comment]
Is there documentation on what to send the nodes as inputs? I'm trying to use the DynamoDB node, do I send the params for a scan as the payload? Am I expected to dig into the Node-RED module code to figure this out? Please make some node documentation with examples, I'll help write it as soon as I get some insight, people who have never used the AWS API will never figure this out. I haven't figured it out and I have written Dynamo queries in Node on Lambda. I would love to help document all of these nodes once I know how to use them in Node-RED. Are there flow examples anywhere?

**Comment by antonio1000homens** [comment]
Yeah same here struggling without examples, not sure what I should put in the "key" of the dynamoDB. Is it msg.payload? payload? a json map?

**Comment by jeremybrun** [comment]
I can confirm that examples are not working. I can't manage to find a way to actually send the data I want in the firehose node. Next step: digging into the source code, but that just sucks.
A response from the maintainer would be appreciated.
